api-schema/src/main/java/org/stellar/anchor/api/sep/sep12/PutCustomerVerificationRequest.java

```diff
@@ -3,8 +3,15 @@
 import java.util.List;
 import lombok.Data;
 
+/**
+ * The request body of the PUT /customer/verification endpoint of SEP-12.
+ *
+ * @see <a
+ *     href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#fields">Refer
+ *     to SEP-12</a>
+ */
 @Data
 public class PutCustomerVerificationRequest {
```

Comment:
Was the request used previously? Wondering why typo hasn't been caught previously

Reply:
This is not used because I don't think Bitso was using it. So we skipped the implementation at that time.